<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutVersionUI</class>
 <widget class="QWidget" name="AboutVersionUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_UpdateDay">
    <property name="text">
     <string>修改日期:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_UpdateDayT">
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget_VersionTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>431</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>版本號</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>修改日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>修改內容</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_Version">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>當前版本:</string>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_VersionT">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
